docs: add quick start (#294)
bytebase/bytebase.com: content/docs/en/get-started/quick-start.md

---
title: Quick Start (5 mins Test)
---

## Overview

> In this guide, you‘ll use **"Bytebase Test Suite"** (one Bytebase and two MySQL instances) to get familiar with the product in the quickest way.

Reviewer: ```suggestion
In this guide, you'll use **"Bytebase Test Suite"** (one Bytebase and two MySQL instances) to get familiar with the product in the quickest way.
```

> In this guide, you‘ll be provided with **"Bytebase Test Suite"** (1 Bytebase and 2 MySQL instances) to get acquainted with the system in the quickest way.

Reviewer 2: ```suggestion
In this guide, you‘ll use **"Bytebase Test Suite"** (1 Bytebase and 2 MySQL instances) to get familiar with the product in the quickest way.
```

Reviewer 2: Try to use more common words

## Prerequisites

Before starting, make sure you have installed Docker. You can click [here](https://www.docker.com/get-started/) to get Docker.

## Procedure

> ### Step 1 Deploy with sample datasets

Reviewer 2: Step 1 - Deploy with sample datasets

  bytebase         | ██╔══██╗╚██╗ ██╔╝╚══██╔══╝██╔════╝██╔══██╗██╔══██╗██╔════╝██╔════╝
  bytebase         | ██████╔╝ ╚████╔╝    ██║   █████╗  ██████╔╝███████║███████╗█████╗
  bytebase         | ██╔══██╗  ╚██╔╝     ██║   ██╔══╝  ██╔══██╗██╔══██║╚════██║██╔══╝
  bytebase         | ██████╔╝   ██║      ██║   ███████╗██████╔╝██║  ██║███████║███████╗
  bytebase         | ╚═════╝    ╚═╝      ╚═╝   ╚══════╝╚═════╝ ╚═╝  ╚═╝╚══════╝╚══════╝
  bytebase         |
  bytebase         | Version 1.2.0 has started at <http://localhost:5678>
  bytebase         | ___________________________________________________________________________________________
  bytebase         |
  employee-test_1  | [Entrypoint] Database initialized
```

Now you have 3 Docker containers running:

- A Bytebase instance
- A MySQL instance for the Test environment
- A MySQL instance for the Prod environment

![3-containers-in-docker](/docs/en/get-started/quick-start/3-containers-in-docker.webp)

Each MySQL instance has a copy of the sample dataset in it. it’s the dataset_small from [open-source database "employee"](https://github.com/bytebase/employee-sample-database-mysql).

> 3. Open Bytebase in [localhost:5678](http://localhost:5678/), if you can see the following page in the browser,  you’re in the right place.

Reviewer 2: ```suggestion
3. Open Bytebase in [localhost:5678](http://localhost:5678/) and you should see the following page in the browser.
```

![welcome-page](/docs/en/get-started/quick-start/welcome-page.webp)

4. Create an admin account, log in, and you are in the workspace.
   

### Step 2 Prepare the Workspace

> #### 2-1 Set Up Environments

Reviewer 2: 2.1

- Click **Environments** on the top bar and you will see **Test** and **Prod** environments. You can keep the default settings or adjust them based on your needs.

Prod environment requires manual approval while Test environment skips that.

#### 2-2 Add Instances

> 1. Click **Instances** on the top bar.

Reviewer 2: This is where most people have trouble (remember how hard we figure out how to connect to an instance?) and we'd better have an image.

2. Click **Add Instance** on the instances page, and you will see **Create Instance** dialog box.
3. Fill the fields as follows:
- Choose **MySQL**
- **Instance Name**: Sample Instance Test
- **Environment**: Test
- **Host or Socket**: host.docker.internal
- **Port**: 3307
- **Username**: root
- **Password**: [empty]
4. Click **Create**, and the instance is created.
5. Roll down to the bottom, you can find a section showing **Databases** and **Users** from this instance.
6. Repeat the operation with another instance and :
- Choose **MySQL**
- **Instance Name**: Sample Instance Prod
- **Environment**: Prod
- **Host or Socket**: host.docker.internal
- **Port**: 3306
- **Username**: root
- **Password**: [empty]

> Now, you have configured 2 instances for Test and Prod environment containing copies of the same sample dataset.

Reviewer 2: Need to have a policy on whether use number `2 `or word `two`

Author: We'll spell the number =<10

> ### Step 3 Run an SQL Review Workflow in a Project

Reviewer 2: People don't understand our jargon. Instead, we should focus on the specifics.

Add a column nickname to the employee table

#### 3-1 Create a project

In Bytebase, **Project** is the unit to contain and manage databases. Therefore, before dealing with the sample databases, you need to transfer them into a project first.

1. Click **Projects** > **New Project** , you will find **Create Project** dialog box.
2. Fill in **Project Name** with Employee, **Key** with T8T, and Select **Mode** as Standard.
3. The "Employee" project is created successfully and you are on its detail page.

![create-a-project](/docs/en/get-started/quick-start/create-a-project.webp)

4. Choose **Transfer DB** to transfer in the 2 "Employee" databases from Test and Prod environment. They’re in the default project since they have not been transferred into any specific project yet.

Now you are ready to manage these 2 "employee" databases in your project.Your next task is to add a column named "nickname" to the "employee" table.

#### 3-2 Create an SQL issue

1. Go to "Employee" project page.
2. Click **Alter Schema**, you will find an **Alter Schema** dialog box.
3. Choose **Alter multiple databases**, select **Test > employee, Prod > employee**, and click **Next**.

> 4. An issue is created, and you will be navigated to the new issue page.

Reviewer 2: Here we can explain a bit about the issue. e.g. there are 2 stages blablabla
